src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1082,36 +1082,35 @@ SPECTRE_ALWAYS_INLINE constexpr void mutate_apply(
       "Cannot pass a DataBox to mutate_apply since the db::get won't work "
```

Comment:
I'll let you take another pass at the commit message. Some grammar things:
- `as it should always Tag::type` -> `as it should always use Tag::type`
- `This was ability was` -> `This ability was`

Reply:
(nitpick, fix only if you get the chance) Typo in commit message: `implementation`